docs/aks-installation.md

```diff
@@ -110,6 +126,103 @@ you should be able to see these Pods running in your cluster:
     pod "tunnelfront-649ff5fb55-5lxg7" deleted
     ```
 
+### Deploying Antrea in `encap` mode
+
+AKS now officially supports [Bring your own Container Network Interface (BYOCNI)](https://learn.microsoft.com/en-us/azure/aks/use-byo-cni).
+Thanks to this, you can deploy Antrea on EKS in `encap` mode, and you will not
+lose access to any functionality. Check the AKS BYOCNI documentation for
+prerequisites, in particular for AKS version requirements.
+
+#### Creating the cluster
+
+You can use any method to create an AKS cluster. The example given here is using the Azure Cloud CLI.
+
+1. Create an AKS Cluster
```

Comment:
I saw the title of this section is "Creating the cluster", but in the detail steps, you used 'Cluster', probably change them to `cluster`.
```suggestion
1. Create an AKS cluster
```